.deployment/deploy-deepforge

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+DEEPFORGE_DEPLOYMENT_DIR="$(realpath "$(dirname "$0")")"
+. ~/.nvm/nvm.sh
+SERVER_NAME=server
+CUSTOM_OVERRIDE_FILE=docker-compose-overrides.yml
```

Comment:
Why is this a variable? It is only used once and its value doesn't seem to ever vary.